support-frontend/assets/stylesheets/emotion/colours.js

```diff
@@ -3,3 +3,5 @@ export const digitalSubscriptionsBlue = '#00568D';
 export const guardianWeeklyBlue = '#66c2e9';
 
 export const paperSubscriptionsBlue = '#90DCFF';
+
+export const comparisonTableYellow = '#FFFACC';
```

Comment:
Nitpick: I'm not sure there's all that much value in defining this in a separate file as it is only used in one place, you could just move it to `tableContents.jsx`

Reply:
Yes, that's fair! 👍🏻